src/pudl_rmi/create_override_spreadsheets.py

```diff
@@ -0,0 +1,646 @@
+    # Enable unique file names and put all files in directory called overrides
+    new_output_path = (
+        pudl_rmi.OUTPUTS_DIR / "overrides" / f"{util_name}_fix_FERC-EIA_overrides.xlsx"
+    )
+    # Output file to a folder called overrides
+    logger.info("Outputing table subsets to tabs\n")
+    writer = pd.ExcelWriter(new_output_path, engine="xlsxwriter")
+    for df_name, df in util_year_subset_dict.items():
+        df.to_excel(writer, sheet_name=df_name, index=False)
+    writer.save()
+
+
+def generate_override_tools(pudl_out, rmi_out, util_dict, years) -> None:
+    """Generate inputs and generate override sheets.
+
+    Args:
+        pudl_out (PudlTabl): the pudl_out object generated in a notebook and passed in.
+        rmi_out (Output): the rmi_out object generated in a notebook and passed in.
+        util_dict (dict): A dictionary with keys that are the names of utility
+            parent companies and values that are lists of subsidiary utility_id_eia
+            values. EIA values are used instead of PUDL in this case because PUDL values
+            are subject to change.
+        years (list): A list of the years you'd like to add to the override sheets.
+
+    """
+    # Generate full input tables
+    inputs_dict = _generate_input_dfs(pudl_out, rmi_out)
+
+    # Make sure overrides dir exists
+    if not os.path.isdir(pudl_rmi.OUTPUTS_DIR / "overrides"):
+        os.mkdir(pudl_rmi.OUTPUTS_DIR / "overrides")
+
+    # For each utility, make an override sheet with the correct input table slices
+    for util_name, util_id_eia_list in util_dict.items():
+        logger.info(f"Developing outputs for {util_name}")
+        util_year_subset_dict = _get_util_year_subsets(
+            inputs_dict, util_id_eia_list, years
+        )
+        _output_override_sheet(util_year_subset_dict, util_name)
+
+
+# --------------------------------------------------------------------------------------
+# Upload Changes to Training Data
+# --------------------------------------------------------------------------------------
+
+
+def _check_id_consistency(id_type, df, actual_ids, error_message):
+    """Check for rogue FERC or EIA ids that don't exist.
+
+    Args:
+        id_type (str): A string, either 'ferc' or 'eia' indicating whether to check
+            ferc or eia id columns.
+        actual_ids (list): A list of the ferc or eia ids that are valid and come from
+            either the ppl or official ferc-eia record linkage.
+        error_message (str): A short string to indicate the type of error you're
+            checking for. This could be looking for values that aren't in the official
+            list or values that are already in the training data.
+
+    """
+    logger.debug(f"Checking {id_type} record id consistency for {error_message}")
+
+    if id_type not in ["ferc", "eia"]:
+        raise ValueError("id_type must be either 'ferc' or 'eia'.")
+    if id_type == "eia":
+        id_col = "record_id_eia_override_1"
```

Comment:
Mm ok fair! I got rid of `id_type` in favor of just `id_col`. For now, I'm keeping it `record_id_eia_override_1` because I don't want to break a bunch of other downstream stuff.